frontend/packages/client/src/components/Community/StrategyEditorModal/StrategySelector.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { kebabToString } from 'utils';
+
+export default function StrategySelector({
+  strategies = [],
```

Comment:
this is coming from backend: 
```
[
    {
        "key": "token-weighted-default",
        "name": "Token Weighted",
        "description": "Number of tokens held directly corresponds to the weight of the vote"
    },
    {
        "key": "staked-token-weighted-default",
        "name": "Staked Token Weighted",
        "description": "Vote weight is proportional to the number tokens staked."
    },
    {
        "key": "one-address-one-vote",
        "name": "One Address One Vote",
        "description": "one address is simply only allowed one vote, assets do not come into play."
    }
]

```
now we are using `kebabToString` function to change the key into a string. `name` field could be also used